src/contour/helper.cpp

```diff
@@ -407,7 +407,7 @@ bool sendKeyEvent(QKeyEvent* event, vtbackend::KeyboardEventType eventType, Term
     if (!event->text().isEmpty())
     {
         auto const codepoints = event->text().toUcs4();
-        assert(codepoints.size() == 1);
+        assert(codepoints.size() >= 1);
```

Comment:
i just noticed now, I am confused. Isn't assert() to only evaluate in debug mode? Our release builds should have that all not generated any code. :thinking:

Reply:
In crispy we have `Require` , this `assert` is from `<cassert>`

Reply:
So this is archlinux specific issue, since debug build is packaged there